internal/overlord/servstate/manager.go

```diff
@@ -115,9 +115,10 @@ func (m *ServiceManager) Plan() (*plan.Plan, error) {
 	return m.plan, nil
 }
 
-// AppendLayer appends the given layer to the plan's layers and updates the
-// layer.Order field to the new order. If a layer with layer.Label already
-// exists, return an error of type *LabelExists.
+// AppendLayer appends the given layer to the plan's layers, updates the
+// layer.Order field to the new order and removes the existing arguments
+// of this layer's services. If a layer with layer.Label already exists,
+// return an error of type *LabelExists.
```

Comment:
This looks like a left over from prior changes, and needs to be reverted (including the spare line below).

Reply:
Right, missed it. Thanks for picking up on it!